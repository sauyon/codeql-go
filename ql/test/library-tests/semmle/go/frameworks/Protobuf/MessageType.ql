import go

from Protobuf::MessageType msgTyp
where msgTyp.(PointerType).getBaseType().hasQualifiedName("codeql-go-tests/protobuf/protos/query", _)
select msgTyp.pp()
