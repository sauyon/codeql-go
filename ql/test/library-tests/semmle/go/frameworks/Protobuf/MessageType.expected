| * Query |
| * QuerySuite |
| * Query_Alert |
